packages/core/src/components/Tooltip/index.tsx

```diff
@@ -177,25 +184,29 @@ export class Tooltip extends React.Component<Props & WithStylesProps, State> {
     };
     const distance = halfNotch + 1;
 
+    const open = hoveringTarget || hoveringTooltip;
+
     return (
-      <span className={cx(styles.container)} ref={this.containerRef}>
-        <div
-          aria-labelledby={labelID}
-          onMouseEnter={this.handleEnter}
-          onMouseLeave={this.handleClose}
-          onMouseDown={this.handleMouseDown}
-          className={cx(!disabled && underlined && styles.underlined)}
-        >
+      <span
+        className={cx(styles.container)}
+        ref={this.containerRef}
+        onMouseEnter={this.handleEnter('Target')}
+        onMouseLeave={this.handleClose('Target')}
+        onMouseDown={this.handleMouseDown}
+      >
+        <div aria-labelledby={labelID} className={cx(!disabled && underlined && styles.underlined)}>
           {children}
         </div>
 
         <div id={labelID} className={cx(styles.offscreen)}>
           {content}
         </div>
 
-        <Overlay open={open} onClose={this.handleClose} noBackground>
+        {open && (
```

Comment:
if this were done with hover & focus styles, it'd be accessible (e.g. the solution that doesn't use `onMouse` events). the tooltip div would be visually hidden until its parent has hover or focus styles activated. since it is a child of the parent, with focus, should still stay open?